packages/terra-base/src/Base.jsx

```diff
@@ -107,6 +105,3 @@ Base.propTypes = propTypes;
 Base.defaultProps = defaultProps;
 
```

Comment:
With these changes, weren't we going to remove the optional locale prop? IMO we make 'en' the default locale since its our fallback

Reply:
[Last we had talked on the issue, the plan was to make the `locale` prop required.](https://github.com/cerner/terra-core/issues/1149#issuecomment-408193958)

Reply:
That means lines 87-93 can be removed right?

Reply:
Yep, good call. Those can be removed.

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2288/commits/8c66e70a2ea75ed114e3a71d338ac81542c04435